spirv_glsl.cpp

```diff
@@ -1041,14 +1047,32 @@ void CompilerGLSL::emit_header()
 		if (execution.flags.get(ExecutionModePostDepthCoverage))
 			inputs.push_back("post_depth_coverage");
 
+		if (execution.flags.get(ExecutionModePixelInterlockOrderedEXT) ||
```

Comment:
Should probably combine this check into a bool so it's not redundantly checked below.